<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1268</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>103</y>
         <width>1061</width>
         <height>311</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="SemanaLayout">
        <item>
         <layout class="QVBoxLayout" name="GeneralLayout" stretch="7,0,0,0,0,0,0">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="cartel_general">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>General</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="lm_grl">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="grl_tm1">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="grl_tm2"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="grl_tm3"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="grl_tm4"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="grl_tm5"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="LunesLayout">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="cartel_lunes">
            <property name="text">
             <string>Lunes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="lm_lu"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="lu_tm1"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="lu_tm2"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="lu_tm3"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="lu_tm4"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="lu_tm5"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="MartesLayout">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="cartel_martes">
            <property name="text">
             <string>Martes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="lm_mt"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="mt_tm1"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="mt_tm2"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="mt_tm3"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="mt_tm4"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="mt_tm5"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="MiercolesLayout">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="cartel_miercoles">
            <property name="text">
             <string>Miercoles</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="lm_mc"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="mc_tm1"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="mc_tm2"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="mc_tm3"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="mc_tm4"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="mc_tm5"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="JuevesLayout">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="cartel_jueves">
            <property name="text">
             <string>Jueves</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="lm_jv"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="jv_tm1"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="jv_tm2"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="jv_tm3"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="jv_tm4"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="jv_tm"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="ViernesLayout">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="cartel_viernes">
            <property name="text">
             <string>Viernes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="lm_vn"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="vn_tm1"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="vn_tm2"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="vn_tm3"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="vn_tm4"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="vn_tm5"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="SabadoLayout">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="cartel_sabado">
            <property name="text">
             <string>Sabado</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="lm_sb"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="sb_tm1"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="sb_tm2"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="sb_tm3"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="sb_tm4"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="sb_tm5"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="DomingoLayout">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="cartel_domingo">
            <property name="text">
             <string>Domingo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="lm_dg"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="dg_tm1"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="dg_tm2"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="dg_tm3"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="dg_tm4"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="dg_tm5"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="volver_boton">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>131</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: blue;  /* Cambia 'green' al color que prefieras */
    color: white;  /* Cambia 'white' al color del texto que prefieras */
border rad
}
QPushButton { border-radius: 15px; /* Ajusta el valor para hacer el botón más o menos redondeado / background-color: #3498db; / Color de fondo del botón / color: white; / Color del texto del botón / border: 12px solid #2980b9; / Borde del botón */ }</string>
       </property>
       <property name="text">
        <string>Volver</string>
       </property>
      </widget>
      <widget class="QPushButton" name="enviar_boton">
       <property name="geometry">
        <rect>
         <x>1020</x>
         <y>420</y>
         <width>131</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: blue;  /* Cambia 'green' al color que prefieras */
    color: white;  /* Cambia 'white' al color del texto que prefieras */
border rad
}
QPushButton { border-radius: 15px; /* Ajusta el valor para hacer el botón más o menos redondeado / background-color: #3498db; / Color de fondo del botón / color: white; / Color del texto del botón / border: 12px solid #2980b9; / Borde del botón */ }</string>
       </property>
       <property name="text">
        <string>Enviar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>0</y>
         <width>1261</width>
         <height>571</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Graficos/2.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <zorder>label</zorder>
      <zorder>horizontalLayoutWidget</zorder>
      <zorder>volver_boton</zorder>
      <zorder>enviar_boton</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>volver_boton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>volver()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enviar_boton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>enviar_dato()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>780</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>641</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>volver()</slot>
  <slot>enviar_dato()</slot>
 </slots>
</ui>
